src/extensions/toolbar.js

```diff
@@ -19,7 +19,9 @@ export class Toolbar {
                 ...editorParams,
                 apiURL: this.instance.get_config('api_host'),
             })
-            this.instance.set_config({ debug: true })
+            localStorage.removeItem('toolbar_disabled')
```

Comment:
- I would consider adjusting the name a bit to make it clear that this is disabled due to a user preference.
- Not a big deal, but I'm wondering why we're doing negative case instead of defaulting to disabled unless you enable it?